ffs/scheduler/astore/astore.go

```diff
@@ -94,6 +94,27 @@ func (s *Store) Remove(c cid.Cid) error {
 // GetRenewable returns all Actions that have CidConfigs that have the Renew flag enabled
 // and should be inspected for Deal renewals.
 func (s *Store) GetRenewable() ([]scheduler.Action, error) {
+	as, err := s.query(func(a scheduler.Action) bool {
+		return a.Cfg.Cold.Enabled && a.Cfg.Cold.Filecoin.Renew.Enabled
+	})
+	if err != nil {
+		return nil, fmt.Errorf("querying for repairable actions: %s", err)
+	}
+	return as, nil
+}
+
+// GetRepairable returns all Actions that have CidConfigs with enabled auto-repair.
+func (s *Store) GetRepairable() ([]scheduler.Action, error) {
```

Comment:
The `scheduler` now has a way to know which _CidConfigs_ are repairable for scheduled-interval repairing.